src/api.jl

```diff
@@ -691,11 +693,17 @@ function list_edbs(ctx::Context, database::AbstractString, engine::AbstractStrin
 end
 
 function _list_models(ctx::Context, database::AbstractString, engine::AbstractString; kw...)
-    tx = Transaction(ctx.region, database, engine, "OPEN"; readonly = true)
-    data = body(tx, _make_list_models_action())
-    rsp = _post(ctx, PATH_TRANSACTION; query = query(tx), body = data, kw...).actions
-    length(rsp) == 0 && return []
-    return rsp[1].result.sources
+    models = []
+    response = exec(ctx, database, engine, "def output:__models__ = rel:catalog:model", kw...);
```

Comment:
Just like in the JS SDK, i think we need to generate a name here for `__models__`, rather than using underscores to "hope" that we avoid collision.

This should be done using something like `out_name = "__models__$(rand(UInt32))"` or something like that, then interpolating in `ouput:$out_name` everywhere.